bigtable-dataflow-import/README.md

```diff
@@ -0,0 +1,57 @@
+# HBase Sequence Files to Cloud Bigtable Over Dataflow
+
+This project supports importing HBase Sequence Files to Google Cloud Bigtable using
+Cloud Dataflow.
+
+## Code Sample
+
+The following code may be used to import a HBase Sequence Files into a bigtable:
+
+```
+  public static void main(String[] args) throws Exception {
+    HBaseImportOptions options =
+        PipelineOptionsFactory.fromArgs(args).withValidation().as(HBaseImportOptions.class);
+    Pipeline p = CloudBigtableIO.initializeForWrite(Pipeline.create(options));
+    p
+        .apply("ReadSequenceFile", Read.from(HBaseImportIO.createSource(options)))
+        .apply("ConvertResultToMutations", HBaseImportIO.transformToMutations())
+        .apply("WriteToTable", CloudBigtableIO.writeToTable(
+            CloudBigtableTableConfiguration.fromCBTOptions(options)));
+    p.run();
+  }
+```
+
+## Command Line Arguments
+
+The code sample in the previous section may be run with the following command line arguments:
+
+```
+    -Xbootclasspath/p:${ALPN-JAR} \
+    --project=... \
+    --runner=... \
+    --stagingLocation=... \
+    --bigtableProjectId=... \
+    --bigtableClusterId=... \
+    --bigtableZoneId=... \
+    --bigtableTableId=... \
+    --filePattern=... \
+```
+
+### ALPN
+
+Follow the [instructions here](https://cloud.google.com/bigtable/docs/installing-hbase-shell#alpn)
+to find out the correct version of alpn for your JRE, then download alpn-boot-${ALPN_VERSION}.jar
+ from the [maven repository](http://central.maven.org/maven2/org/mortbay/jetty/alpn/alpn-boot).
+Replace ${ALPN-JAR} above with the path of the downloaded jar.
+
+### More on Arguments
+The runner argument specifies where the job is run. If the input files are on local file system,
```

Comment:
As soon as this is released, we'll add an example and reference it from this file. 
I don't want to duplicate the pom.xml here.